docs/design.md

```diff
@@ -191,7 +363,8 @@ kind: Notebook
 name: notebook-training-experiment
 spec:
   image:
-    upload: {} # This is how the plugin signals it wants to upload a directory for building.
+    upload:
+      checksum: 11ddbaf3386aea1f2974eee984542152 # This is how the plugin signals it wants to upload a directory for building.
```

Comment:
would this be the checksum of the tar file?

Reply:
yeah, related issue: https://github.com/substratusai/substratus/issues/95